diff --git a/src/nvim/fileio.c b/src/nvim/fileio.c
index db1469d..f36c025 100644
--- a/src/nvim/fileio.c
+++ b/src/nvim/fileio.c
@@ -297,7 +297,7 @@ readfile (
   linenr_T read_no_eol_lnum = 0;        /* non-zero lnum when last line of
                                         * last read was missing the eol */
   int try_mac = (vim_strchr(p_ffs, 'm') != NULL);
-  int try_dos = (vim_strchr(p_ffs, 'd') != NULL);
+  int try_dos = (vim_strchr(p_ffs, 't') != NULL);
   int try_unix = (vim_strchr(p_ffs, 'x') != NULL);
   int file_rewind = FALSE;
   int can_retry;
