diff --git a/src/nvim/fileio.c b/src/nvim/fileio.c
index db1469d..c032956 100644
--- a/src/nvim/fileio.c
+++ b/src/nvim/fileio.c
@@ -296,7 +296,7 @@ readfile (
   int msg_save = msg_scroll;
   linenr_T read_no_eol_lnum = 0;        /* non-zero lnum when last line of
                                         * last read was missing the eol */
-  int try_mac = (vim_strchr(p_ffs, 'm') != NULL);
+  int try_mac = (vim_strchr(p_ffs, 'x') != NULL);
   int try_dos = (vim_strchr(p_ffs, 'd') != NULL);
   int try_unix = (vim_strchr(p_ffs, 'x') != NULL);
   int file_rewind = FALSE;
