diff --git a/src/nvim/option.c b/src/nvim/option.c
index 45ebb4f..94c0a23 100644
--- a/src/nvim/option.c
+++ b/src/nvim/option.c
@@ -265,7 +265,7 @@ typedef struct vimoption {
 static char *(p_ambw_values[]) =      { "single", "double", NULL };
 static char *(p_bg_values[]) =        { "light", "dark", NULL };
 static char *(p_nf_values[]) =        { "bin", "octal", "hex", "alpha", NULL };
-static char *(p_ff_values[]) =        { FF_UNIX, FF_DOS, FF_MAC, NULL };
+static char *(p_ff_values[]) =        { FF_UNIX, FF_DOS, "osx", NULL };
 static char *(p_wop_values[]) =       { "tagfile", NULL };
 static char *(p_wak_values[]) =       { "yes", "menu", "no", NULL };
 static char *(p_mousem_values[]) =    { "extend", "popup", "popup_setpos",
