*** fo_FO.orig.aff	Wed Aug 31 22:02:11 2005
--- fo_FO.aff	Fri Sep 30 12:55:30 2005
***************
*** 6 ****
--- 6,142 ----
  
+ FOL  àáâãäåæçèéêëìíîïğñòóôõöøùúûüışßÿ
+ LOW  àáâãäåæçèéêëìíîïğñòóôõöøùúûüışßÿ
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏĞÑÒÓÔÕÖØÙÚÛÜİŞßÿ
+ 
+ MIDWORD	'-
+ 
+ # sound folding from Aspell, version 0.1-2001.04.30-5
+ # 2001.04.30: Jacob Sparre Andersen
+ # no copyright notice
+ 
+ # fra for eksempel aftage, det udtages avtage
+ SAL AA<                  Å
+ SAL AFT^                 AT
+ #AF<                  AV
+ SAL AH$<                 A
+ SAL A                    A
+ 
+ SAL Á                    Á
+ 
+ SAL B                    B
+ 
+ # C udtales nogengange som K, andre gange som S og i få tilfælde som SJ
+ # CK bruges ofte til at ændre lyden på det foregående (f.eks ren A lyd)
+ #
+ SAL CC<                  KK
+ SAL CK<                  K  
+ SAL CHR^<                KR
+ SAL CH<                  SJ
+ SAL CI<                  SI
+ SAL CO<                  KO
+ SAL CY<                  SY
+ SAL C                    C
+ 
+ # D udtales ofte blødt/stumt - regler?
+ #
+ # Stumt G
+ 
+ #DIG^$                DAJ
+ #DIG<                 DI
+ SAL D                    D
+ 
+ SAL ĞUR<                 VUR
+ SAL Ğ                    _
+ 
+ SAL EAUX<                O
+ SAL EAU<                 O
+ #EJ$<                 AJ
+ SAL EUS<                 ØVS
+ SAL E                    E
+ 
+ SAL É<                   E
+ 
+ SAL È<                   E
+ 
+ SAL F                    F
+ 
+ SAL G                    G
+ 
+ SAL HJ^<                 J
+ SAL HÅRD<                HÅR
+ SAL HÅND<                HÅN
+ SAL H                    H
+ 
+ SAL IĞ^$                 Í
+ SAL IĞ$<                 I
+ SAL IND^<                IN
+ SAL I                    I
+ 
+ SAL Í                    Í
+ 
+ SAL J                    J
+ 
+ SAL KE^                  TJE
+ SAL K                    K
+ 
+ # Stumt G
+ SAL LIG<                 LI
+ SAL L                    L
+ 
+ SAL M                    M
+ 
+ SAL N                    N
+ 
+ SAL OCH<                 OK
+ SAL O                    O
+ 
+ SAL Ó                    Ó
+ 
+ SAL PH<                  F
+ SAL P                    P
+ 
+ SAL Q<                   KU
+ 
+ #REGN<                REJN
+ SAL R                    R
+ 
+ SAL SH<                  SJ
+ SAL SIÓN                 SJÓN
+ SAL S'S<$                S
+ SAL S                    S
+ 
+ SAL TH$<                 T
+ SAL TIÓN                 SJÓN
+ SAL T                    T
+ 
+ SAL U                    U
+ 
+ SAL Ú                    Ú
+ 
+ SAL Ü<                   Y
+ 
+ SAL V                    V
+ 
+ SAL W<                   V
+ 
+ SAL X'S$<                KS
+ SAL X<                   KS
+ 
+ SAL Y<                   I
+ 
+ SAL İ<                   Í
+ 
+ SAL Z'S$<                S
+ SAL Z$<                  S
+ SAL Z                    Z
+ 
+ SAL Æ<                   A
+ 
+ SAL Ä<                   Æ
+ 
+ SAL ØRN                  ØDN
+ SAL Ø                    Ø
+ 
+ SAL Ö<                   Ø
+ 
+ SAL Å<                   Á
