From 69313032fad04743c96bc8f2a029b691857488f9 Mon Sep 17 00:00:00 2001
Date: Thu, 1 Mar 2018 21:41:29 -0600
Subject: [PATCH] Change default downloader to curl

---
 install.bat | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install.bat b/install.bat
index 09cf9aa..76e4059 100644
--- a/install.bat
+++ b/install.bat
@@ -1037,7 +1037,7 @@ f:write(S[=[
 site_config.LUAROCKS_UNAME_M=[[$UNAME_M]]
 site_config.LUAROCKS_ROCKS_TREE=[[$TREE_ROOT]]
 site_config.LUAROCKS_PREFIX=[[$PREFIX]]
-site_config.LUAROCKS_DOWNLOADER=[[wget]]
+site_config.LUAROCKS_DOWNLOADER=[[curl]]
 site_config.LUAROCKS_MD5CHECKER=[[md5sum]]
 ]=])
 if FORCE_CONFIG then
-- 
2.16.1.windows.4

