*** en_US.orig.aff	Fri Apr 15 13:20:36 2005
--- en_US.aff	Wed Jan 11 22:03:34 2006
***************
*** 3,4 ****
--- 3,141 ----
  
+ FOL  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ LOW  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ‗ÿ
+ 
+ MIDWORD	'
+ 
+ RARE ?
+ BAD !
+ 
+ MAP 9
+ MAP aאבגדהו
+ MAP eטיךכ
+ MAP iלםמן
+ MAP oעףפץצ
+ MAP uשתûü
+ MAP nס
+ MAP cח
+ MAP yÿ‎
+ MAP s‗
+ 
+ # This comes from Aspell en_phonet.dat, version 1.1, 2000-01-07
+ 
+ SAL AH(AEIOUY)-^         *H
+ SAL AR(AEIOUY)-^         *R
+ SAL A(HR)^               *
+ SAL A^                   *
+ SAL AH(AEIOUY)-          H
+ SAL AR(AEIOUY)-          R
+ SAL A(HR)                _
+ SAL À^                   *
+ SAL Å^                   *
+ SAL BB-                  _
+ SAL B                    B
+ SAL CQ-                  _
+ SAL CIA                  X
+ SAL CH                   X
+ SAL C(EIY)-              S
+ SAL CK                   K
+ SAL COUGH^               KF
+ SAL CC<                  C
+ SAL C                    K
+ SAL DG(EIY)              K
+ SAL DD-                  _
+ SAL D                    T
+ SAL É<                   E
+ SAL EH(AEIOUY)-^         *H
+ SAL ER(AEIOUY)-^         *R
+ SAL E(HR)^               *
+ SAL ENOUGH^$             *NF
+ SAL E^                   *
+ SAL EH(AEIOUY)-          H
+ SAL ER(AEIOUY)-          R
+ SAL E(HR)                _
+ SAL FF-                  _
+ SAL F                    F
+ SAL GN^                  N
+ SAL GN$                  N
+ SAL GNS$                 NS
+ SAL GNED$                N
+ SAL GH(AEIOUY)-          K
+ SAL GH                   _
+ SAL GG9                  K
+ SAL G                    K
+ SAL H                    H
+ SAL IH(AEIOUY)-^         *H
+ SAL IR(AEIOUY)-^         *R
+ SAL I(HR)^               *
+ SAL I^                   *
+ SAL ING6                 N
+ SAL IH(AEIOUY)-          H
+ SAL IR(AEIOUY)-          R
+ SAL I(HR)                _
+ SAL J                    K
+ SAL KN^                  N
+ SAL KK-                  _
+ SAL K                    K
+ SAL LAUGH^               LF
+ SAL LL-                  _
+ SAL L                    L
+ SAL MB$                  M
+ SAL MM                   M
+ SAL M                    M
+ SAL NN-                  _
+ SAL N                    N
+ SAL OH(AEIOUY)-^         *H
+ SAL OR(AEIOUY)-^         *R
+ SAL O(HR)^               *
+ SAL O^                   *
+ SAL OH(AEIOUY)-          H
+ SAL OR(AEIOUY)-          R
+ SAL O(HR)                _
+ SAL PH                   F
+ SAL PN^                  N
+ SAL PP-                  _
+ SAL P                    P
+ SAL Q                    K
+ SAL RH^                  R
+ SAL ROUGH^               RF
+ SAL RR-                  _
+ SAL R                    R
+ SAL SCH(EOU)-            SK
+ SAL SC(IEY)-             S
+ SAL SH                   X
+ SAL SI(AO)-              X
+ SAL SS-                  _
+ SAL S                    S
+ SAL TI(AO)-              X
+ SAL TH                   @
+ SAL TCH--                _
+ SAL TOUGH^               TF
+ SAL TT-                  _
+ SAL T                    T
+ SAL UH(AEIOUY)-^         *H
+ SAL UR(AEIOUY)-^         *R
+ SAL U(HR)^               *
+ SAL U^                   *
+ SAL UH(AEIOUY)-          H
+ SAL UR(AEIOUY)-          R
+ SAL U(HR)                _
+ SAL V^                   W
+ SAL V                    F
+ SAL WR^                  R
+ SAL WH^                  W
+ SAL W(AEIOU)-            W
+ SAL X^                   S
+ SAL X                    KS
+ SAL Y(AEIOU)-            Y
+ SAL ZZ-                  _
+ SAL Z                    S
+ 
+ # When soundfolding "th" is turned into "@".  When this is mistyped as "ht" it
+ # soundfolds to "ht".  This difference is too big, thus use REP items to lower
+ # the score.
+ REPSAL 2
+ REPSAL ht @
+ REPSAL @ ht
+ 
  PFX A Y 1
***************
*** 30,33 ****
  SFX N   e     ion        e
! SFX N   y     ication    y 
! SFX N   0     en         [^ey] 
  
--- 167,170 ----
  SFX N   e     ion        e
! SFX N   y     ication    y
! SFX N   0     en         [^ey]
  
***************
*** 40,42 ****
  SFX H   y     ieth       y
! SFX H   0     th         [^y] 
  
--- 177,179 ----
  SFX H   y     ieth       y
! SFX H   0     th         [^y]
  
***************
*** 47,49 ****
  SFX G   e     ing        e
! SFX G   0     ing        [^e] 
  
--- 184,186 ----
  SFX G   e     ing        e
! SFX G   0     ing        [^e]
  
***************
*** 99,101 ****
  
! REP 88
  REP a ei
--- 236,238 ----
  
! REP 91
  REP a ei
***************
*** 137,138 ****
--- 274,277 ----
  REP uy i
+ REP y ie
+ REP ie y
  REP i ee
***************
*** 174,175 ****
--- 313,315 ----
  REP ew ue
+ REP uf ough
  REP uff ough
***************
*** 188 ****
--- 328,332 ----
  REP shun cion
+ REP an_a a
+ REP an_a an
+ REP a_an a
+ REP a_an an
*** en_US.orig.dic	Fri Apr 15 13:20:36 2005
--- en_US.dic	Wed Mar  8 13:14:43 2006
***************
*** 1,2 ****
! 62076
  a
--- 1,2 ----
! 62078
  a
***************
*** 5944,5946 ****
  bk
! b/KGD
  Bk/M
--- 5944,5947 ----
  bk
! probing
! probed
  Bk/M
***************
*** 9007,9009 ****
  Cazzie/M
- c/B
  CB
--- 9008,9009 ----
***************
*** 9233,9235 ****
  cetacean/S
- cetera/S
  Cetus/M
--- 9233,9234 ----
***************
*** 11575,11576 ****
--- 11574,11577 ----
  conduit/MS
+ coned
+ cone/MS
  coneflower/M
***************
*** 11712,11713 ****
--- 11713,11715 ----
  coniferous
+ conj.
  conjectural/Y
***************
*** 14038,14043 ****
  dazzling/Y
- db
- DB
  dbl
  dB/M
  DBMS
--- 14040,14046 ----
  dazzling/Y
  dbl
  dB/M
+ dBi
+ dBm
+ dBd
  DBMS
***************
*** 15464,15466 ****
  dingbat/MS
! ding/GD
  dinghy/SM
--- 15467,15469 ----
  dingbat/MS
! ding/GDS
  dinghy/SM
***************
*** 15690,15692 ****
  dishevelment/MS
! dish/GD
  dishonest
--- 15693,15695 ----
  dishevelment/MS
! dish/GDMS
  dishonest
***************
*** 15973,15975 ****
  djellaba/S
- d/JGVX
  Djibouti/M
--- 15976,15977 ----
***************
*** 16911,16912 ****
--- 16913,16915 ----
  dusty/RPT
+ Farsi
  Dutch/M
***************
*** 17357,17359 ****
  EFL
- e/FMDS
  Efrain/M
--- 17360,17361 ----
***************
*** 18780,18782 ****
  estuary/SM
! et
  ET
--- 18782,18785 ----
  estuary/SM
! et cetera/S
! et al.
  ET
***************
*** 18785,18787 ****
  eta/SM
! etc
  etcetera/SM
--- 18788,18790 ----
  eta/SM
! etc.
  etcetera/SM
***************
*** 20559,20561 ****
  Fiori/M
- f/IRAC
  firearm/SM
--- 20562,20563 ----
***************
*** 24402,24404 ****
  guzzler/M
! g/VBX
  Gwalior/M
--- 24404,24406 ----
  guzzler/M
! gens
  Gwalior/M
***************
*** 25473,25475 ****
  hemp/MNS
- h/EMS
  hemstitch/DSMG
--- 25475,25476 ----
***************
*** 25963,25965 ****
  hobbing
! hobbit
  hobbler/M
--- 25964,25966 ----
  hobbing
! hobbit/MS
  hobbler/M
***************
*** 26524,26526 ****
  HST
- ht
  HTML
--- 26525,26526 ----
***************
*** 26942,26944 ****
  Hz
- i
  I
--- 26942,26943 ----
***************
*** 29627,29629 ****
  Jezebel/MS
- j/F
  JFK/M
--- 29626,29627 ----
***************
*** 30578,30580 ****
  keyword/SM
! k/FGEIS
  kg
--- 30576,30583 ----
  keyword/SM
! inking
! disking
! conking
! inks
! disks
! conks
  kg
***************
*** 32694,32696 ****
  Lizzy/M
! l/JGVXT
  Ljubljana/M
--- 32697,32699 ----
  Lizzy/M
! lings
  Ljubljana/M
***************
*** 34456,34458 ****
  mash/JGZMSRD
! m/ASK
  masked/U
--- 34459,34462 ----
  mash/JGZMSRD
! rems
! prom/S
  masked/U
***************
*** 34746,34747 ****
--- 34750,34753 ----
  Mb
+ Mbyte
+ Mbit
  MB
***************
*** 36605,36606 ****
--- 36611,36613 ----
  Moog
+ Moolenaar/M
  moo/GSD
***************
*** 38871,38873 ****
  NSF
- n/T
  NT
--- 38878,38879 ----
***************
*** 39011,39013 ****
  NZ
- o
  O
--- 39017,39018 ----
***************
*** 39532,39534 ****
  om/XN
- ON
  onanism/M
--- 39537,39538 ----
***************
*** 42508,42510 ****
  pinfeather/SM
! ping/GDRM
  pinheaded/P
--- 42512,42514 ----
  pinfeather/SM
! ping/GDRMS
  pinheaded/P
***************
*** 42983,42984 ****
--- 42987,42989 ----
  pneumonia/MS
+ pneumonic
  PO
***************
*** 43216,43218 ****
  pompom/SM
! pompon's
  pomposity/MS
--- 43221,43223 ----
  pompom/SM
! pompon/M
  pomposity/MS
***************
*** 44940,44942 ****
  PX
- p/XTGJ
  Pygmalion/M
--- 44945,44946 ----
***************
*** 44983,44985 ****
  pyx/MDSG
- q
  Q
--- 44987,44988 ----
***************
*** 46507,46509 ****
  Renault/MS
- rend
  renderer/M
--- 46510,46511 ----
***************
*** 47258,47260 ****
  ringer/M
! ring/GZJDRM
  ringing/Y
--- 47260,47262 ----
  ringer/M
! ring/GZJDRMS
  ringing/Y
***************
*** 47857,47862 ****
  rt
- rte
  Rte
  RTFM
- r/TGVJ
  Rubaiyat/M
--- 47859,47862 ----
***************
*** 48085,48087 ****
  Ryun/M
- S
  SA
--- 48085,48086 ----
***************
*** 54450,54452 ****
  swung
! s/XJBG
  sybarite/MS
--- 54449,54451 ----
  swung
! sings
  sybarite/MS
***************
*** 56906,56908 ****
  transit/SGVMD
! transl
  translatability/M
--- 56905,56907 ----
  transit/SGVMD
! transl.
  translatability/M
***************
*** 57728,57730 ****
  TX
! t/XTJBG
  Tybalt/M
--- 57727,57729 ----
  TX
! tings
  Tybalt/M
***************
*** 57809,57811 ****
  Tzeltal/M
- u
  U
--- 57808,57809 ----
***************
*** 58494,58495 ****
--- 58492,58494 ----
  unsearchable
+ searchable
  unseasonal
***************
*** 59072,59074 ****
  vast/PTSYR
! v/ASV
  VAT
--- 59071,59073 ----
  vast/PTSYR
! revs
  VAT
***************
*** 59538,59540 ****
  vi/MDR
! vim/MS
  vinaigrette/MS
--- 59537,59540 ----
  vi/MDR
! Vim/MS
! vim/?
  vinaigrette/MS
***************
*** 61534,61536 ****
  WWW
! w/XTJGV
  WY
--- 61534,61537 ----
  WWW
! wens
! wings
  WY
***************
*** 61750,61752 ****
  yew/SM
- y/F
  Yggdrasil/M
--- 61751,61752 ----
***************
*** 62058,62060 ****
  Zsigmondy/M
! z/TGJ
  Zubenelgenubi/M
--- 62058,62060 ----
  Zsigmondy/M
! zings
  Zubenelgenubi/M
***************
*** 62077 ****
--- 62077,62092 ----
  zymurgy/S
+ nd
+ the the/!
+ and and/!
+ a a/!
+ an an/!
+ a an/!
+ an a/!
+ the a/!
+ the an/!
+ a the/!
+ an the/!
+ PayPal
+ Google
+ e.g.
+ TCP\/IP
