*** nb_NO.orig.aff	Wed Aug 31 18:29:43 2005
--- nb_NO.aff	Wed Aug 31 18:35:09 2005
***************
*** 7,8 ****
--- 7,26 ----
  
+ FOL  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ LOW  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ UPP  ‗ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞÿ
+ 
+ SOFOFROM abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ¿
+ SOFOTO   ebctefghejklnnepkrstevvkesebctefghejklnnepkrstevvkeseeeeeeeceeeeeeeedneeeeeeeeeeepseeeeeeeeceeeeeeeedneeeeeeeeeeep?
+ 
+ MAP 9
+ MAP aאבגדהו
+ MAP eטיךכ
+ MAP iלםמן
+ MAP oעףפץצ
+ MAP uשתûü
+ MAP nס
+ MAP cח
+ MAP yÿ‎
+ MAP s‗
+ 
  PFX a Y 1
*** nb_NO.orig.dic	Wed Aug 31 18:29:43 2005
--- nb_NO.dic	Wed Aug 31 18:38:02 2005
***************
*** 2,4 ****
  a.a
- a.a
  a.a.C
--- 2,3 ----
***************
*** 15054,15056 ****
  cand
- cand/
  cand.act
--- 15053,15054 ----
***************
*** 28532,28534 ****
  f.o.r
- fפr
  fora/G
--- 28530,28531 ----
***************
*** 28980,28982 ****
  fordרyelsessystem/BCEFGH
- fפre
  fפre/BEJtz
--- 28977,28978 ----
***************
*** 43532,43534 ****
  Idar/J
- idי
  idי/AEFGH[z
--- 43528,43529 ----
***************
*** 57490,57492 ****
  Lambertseter/J
- lamי
  lamי/A
--- 57485,57486 ----
