*** ro_RO.orig.aff	Wed Aug 31 20:34:38 2005
--- ro_RO.aff	Wed Aug 31 20:39:57 2005
***************
*** 3,4 ****
--- 3,8 ----
  
+ FOL ±Ά³µ¶¨ΉΊ»ΌΎΏ±²³΄µ¶·ΈΉΊ»Ό½ΎΏΰαβγδεζηθικλμνξοπρςστυφψωϊϋόύώίÿ
+ LOW ±Ά³µ¶¨ΉΊ»ΌΎΏ±²³΄µ¶·ΈΉΊ»Ό½ΎΏΰαβγδεζηθικλμνξοπρςστυφψωϊϋόύώίÿ
+ UPP ΅Ά£¥¦¨©ª«¬®―±²³΄µ¶·ΈΉΊ»Ό½ΎΏΐΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡÒΣΤΥΦΨΩΪΫάέήίÿ
+ 
  PFX E Y 1
***************
*** 12,15 ****
  SFX L   0          l          u
! SFX L   0          le         [^cg] i
! SFX L   0          i          [cg] i
  SFX L   0          le         e
--- 16,19 ----
  SFX L   0          l          u
! SFX L   0          le         [^cg]i
! SFX L   0          i          [cg]i
  SFX L   0          le         e
***************
*** 18,20 ****
  SFX U   0          a          re
! SFX U   0          i          [^i] ii
  
--- 22,24 ----
  SFX U   0          a          re
! SFX U   0          i          [^i]ii
  
***************
*** 38,41 ****
  SFX I   0          ului       [^ua]
! SFX I   a          ii         [gc] a
! SFX I   a          ei         [^cg] a
  
--- 42,45 ----
  SFX I   0          ului       [^ua]
! SFX I   a          ii         [gc]a
! SFX I   a          ei         [^cg]a
  
