*** tet_ID.orig.aff	Wed Aug 31 21:14:37 2005
--- tet_ID.aff	Wed Aug 31 21:15:15 2005
***************
*** 19 ****
--- 19,39 ----
  TRY aineousrthlkmdbp'fTvgzLAINjSPEMיD-KHבףFRUBGJתOcםwxCWXVסדÁqךחZÓQyגÍ
+ 
+ FOL  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ LOW  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ UPP  ‗ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞÿ
+ 
+ SOFOFROM abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ¿
+ SOFOTO   ebctefghejklnnepkrstevvkesebctefghejklnnepkrstevvkeseeeeeeeceeeeeeeedneeeeeeeeeeepseeeeeeeeceeeeeeeedneeeeeeeeeeep?
+ 
+ MIDWORD	'-
+ 
+ MAP 9
+ MAP aאבגדהו
+ MAP eטיךכ
+ MAP iלםמן
+ MAP oעףפץצ
+ MAP uשתûü
+ MAP nס
+ MAP cח
+ MAP yÿ‎
+ MAP s‗
