*** hu_HU.orig.aff	Wed Apr 12 12:33:23 2006
--- hu_HU.aff	Wed Apr 12 12:43:39 2006
***************
*** 58,60 ****
  NAME Magyar Ispell helyesírási szótár
! LANG hu_HU
  HOME http://magyarispell.sourceforge.net
--- 58,60 ----
  NAME Magyar Ispell helyesírási szótár
! #LANG hu_HU
  HOME http://magyarispell.sourceforge.net
***************
*** 64,77 ****
  COMPOUNDMIN 2
! COMPOUNDFLAG Y
! COMPOUNDWORD 2 y
! COMPOUNDSYLLABLE 6 aáeéiíoóöõuúüû
! SYLLABLENUM klmc
! COMPOUNDFIRST v
! COMPOUNDLAST x
! FORBIDDENWORD w
! ONLYROOT u
! ACCENT áéíóöõúüû aeiooouuu
! CHECKNUM
! WORDCHARS -.§%°0123456789
! HU_KOTOHANGZO Z
  
--- 64,115 ----
  COMPOUNDMIN 2
! #COMPOUNDWORD 2 y
! COMPOUNDWORDMAX 2
! COMPOUNDROOT y
! 
! #COMPOUNDSYLLABLE 6 aáeéiíoóöõuúüû
! COMPOUNDSYLMAX 6
! SYLLABLE a/á/e/é/i/í/o/ó/ö/õ/u/ú/ü/û
! # Strange that every vowel is counted as a syllable, that's how the hunspell
! # code works.
! 
! # the effect of SYLLABLENUM is not documented
! #SYLLABLENUM klmc
! 
! #COMPOUNDFLAG Y
! #COMPOUNDFIRST v
! #COMPOUNDLAST x
! COMPOUNDRULE Y+
! COMPOUNDRULE vY*x
! COMPOUNDRULE Y+x
! COMPOUNDRULE vY+
! 
! #FORBIDDENWORD w
! BAD w
! 
! #ONLYROOT u
! NEEDAFFIX u
! 
! #ACCENT áéíóöõúüû aeiooouuu
! MAP 5
! MAP aáä
! MAP eé
! MAP ií
! MAP oóöõ
! MAP uúüû
! 
! #CHECKNUM
! # Vim always handles numbers in the same way.
! 
! #WORDCHARS -.§%°0123456789
! FOL ±¢³µ¶¨¹º»¼¾¿±²³´µ¶·¸¹º»¼½¾¿àáâãäåæçèéêëìíîïðñòóôõöøùúûüýþßÿ-§%°
! LOW ±¢³µ¶¨¹º»¼¾¿±²³´µ¶·¸¹º»¼½¾¿àáâãäåæçèéêëìíîïðñòóôõöøùúûüýþßÿ-§%°
! UPP ¡¢£¥¦¨©ª«¬®¯±²³´µ¶·¸¹º»¼½¾¿ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞßÿ-§%°
! MIDWORD .
! 
! # Undocumented
! #HU_KOTOHANGZO Z
! 
! # There are soooo many affixes.  Postpone the prefixes to keep the time needed
! # for generating the .spl within reasonable limits.
! PFXPOSTPONE
  
***************
*** 81,96 ****
  
! REP 89
! REP í i
! REP i í
! REP ó o
! REP o ó
! REP o õ
! REP ú u
! REP u ú
! REP u û
! REP û ü
! REP ü û
  REP j ly
  REP ly j
- REP a ä # Handel->Händel
  REP S © # Skoda->©koda
--- 119,123 ----
  
! REP 78
  REP j ly
  REP ly j
  REP S © # Skoda->©koda
***************
*** 173,241 ****
  
- # character conversion table
- # (HTML latin-1 entities -> latin-2)
- # not implemented yet
- 
- CHR HTML 35
- CHR HTML &curren; ¤
- CHR HTML &deg; °
- CHR HTML &acute; ´
- CHR HTML &cedil; ¸
- CHR HTML &Aacute; Á
- CHR HTML &Acirc; Â
- CHR HTML &Auml; Ä
- CHR HTML &Ccedil; Ç
- CHR HTML &Eacute; É
- CHR HTML &Euml; Ë
- CHR HTML &Iacute; Í
- CHR HTML &Icirc; Î
- CHR HTML &Oacute; Ó
- CHR HTML &Ocirc; Ô
- CHR HTML &Ouml; Ö
- CHR HTML &times; ×
- CHR HTML &Uacute; Ú
- CHR HTML &Uuml; Ü
- CHR HTML &Yacute; Ý
- CHR HTML &szlig; ß
- CHR HTML &aacute; á
- CHR HTML &acirc; â
- CHR HTML &auml; ä
- CHR HTML &ccedil; ç
- CHR HTML &eacute; é
- CHR HTML &euml; ë
- CHR HTML &iacute; í
- CHR HTML &icirc; î
- CHR HTML &oacute; ó
- CHR HTML &ocirc; ô
- CHR HTML &ouml; ö
- CHR HTML &divide; ÷
- CHR HTML &uacute; ú
- CHR HTML &uuml; ü
- CHR HTML &yacute; ý
- 
- # character conversion table
- # (Prószéky-code -> latin-2)
- # not implemented yet
- 
- CHR 123 20
- CHR 123 a1 á
- CHR 123 e1 é
- CHR 123 e2 ë
- CHR 123 i1 í
- CHR 123 o1 ó
- CHR 123 o2 ö
- CHR 123 o3 õ
- CHR 123 u1 ú
- CHR 123 u2 ü
- CHR 123 u3 û
- CHR 123 A1 Á
- CHR 123 E1 É
- CHR 123 E2 Ë
- CHR 123 I1 Í
- CHR 123 O1 Ó
- CHR 123 O2 Ö
- CHR 123 O3 Õ
- CHR 123 U1 Ú
- CHR 123 U2 Ü
- CHR 123 U3 Û
- 
  SFX z Y 6
--- 200,201 ----
***************
*** 17678,17681 ****
  PFX D   0    leg          .
- 
-    1
- 
--- 17638 ----
