*** bg_BG.orig.aff	Sun Aug 28 21:34:44 2005
--- bg_BG.aff	Thu Sep 29 21:59:31 2005
***************
*** 1,2 ****
! SET microsoft-cp1251
  TRY ЮЕМНХПРБЯКЙЛОГДЪСЗАВЦЖФУЬТИЫЧЭюемнхпрбякйлогдъсзавцжфуьтиычэ
--- 1,2 ----
! SET cp1251
  TRY ЮЕМНХПРБЯКЙЛОГДЪСЗАВЦЖФУЬТИЫЧЭюемнхпрбякйлогдъсзавцжфуьтиычэ
***************
*** 1682,1684 ****
  
! MAP 26
  MAP ЮЗЭa
--- 1682,1684 ----
  
! MAP 25
  MAP ЮЗЭa
***************
*** 1691,1695 ****
  MAP ФЬ
! MAP ГЯ
  MAP pП
- MAP cЯ
  MAP xУ
--- 1691,1694 ----
  MAP ФЬ
! MAP cГЯ
  MAP pП
  MAP xУ
***************
*** 1707,1709 ****
  MAP Pп
- MAP Yс
  MAP Xу
--- 1706,1855 ----
  MAP Pп
  MAP Xу
+ 
+ REP 2
+ REP Y с
+ REP с Y
+ 
+ #   Bulgarian phonetic transformation rules for use with Aspell
+ #   Copyright (C) 2003 Anton Zinoviev
+ #
+ #   This software may be used and distributed under the same terms as
+ #   the other parts of the bgoffice project.
+ #
+ #   Changelog:
+ #
+ #   19.IX.2003  Anton Zinoviev <zinoviev@debian.org>
+ #               Initial release
+ 
+ SAL version         1
+ SAL followup        0
+ SAL collapse_result 1
+ 
+ SAL ю                           з
+ SAL з                           з
+ SAL н                           с
+ SAL с                           с
+ SAL е                           х
+ SAL х                           х
+ SAL ч                           с
+ SAL ъ                           з
+ SAL и                           _
+ SAL э                           _
+ SAL аяй                         ой
+ SAL ад(абцдфгйклмопярцужвьы)-   о
+ SAL аT(абцдфгйклмопярцужвьы)-   о
+ SAL ад$                         о
+ SAL аT$                         о
+ SAL а                           о
+ SAL бяй                         тй
+ SAL бд(абцдфгйклмопярцужвьы)-   т
+ SAL бT(абцдфгйклмопярцужвьы)-   т
+ SAL бд$                         т
+ SAL бT$                         т
+ SAL б                           т
+ SAL цяй                         й
+ SAL цд(абцдфгйклмопярцужвьы)-   й
+ SAL цT(абцдфгйклмопярцужвьы)-   й
+ SAL цд$                         й
+ SAL цT$                         й
+ SAL ц                           й
+ SAL дяй<                        рй
+ SAL дд(абцдфгйклмопярцужвьы)-   р
+ SAL дT(абцдфгйклмопярцужвьы)-   р
+ SAL дд$                         р
+ SAL дT$                         р
+ SAL д                           р
+ SAL фяй                         ьй
+ SAL фд(абцдфгйклмопярцужвьы)-   ь
+ SAL фT(абцдфгйклмопярцужвьы)-   ь
+ SAL фд$                         ь
+ SAL фT$                         ь
+ SAL ф                           ь
+ SAL гяй                         яй
+ SAL гд(абцдфгйклмопярцужвьы)-   я
+ SAL гT(абцдфгйклмопярцужвьы)-   я
+ SAL гд$                         я
+ SAL гT$                         я
+ SAL г                           я
+ SAL йяй                         й
+ SAL йд(абцдфгйклмопярцужвьы)-   й
+ SAL йT(абцдфгйклмопярцужвьы)-   й
+ SAL йд$                         й
+ SAL йT$                         й
+ SAL й                           й
+ SAL кяй                         кй
+ SAL кд(абцдфгйклмопярцужвьы)-   к
+ SAL кT(абцдфгйклмопярцужвьы)-   к
+ SAL кд$                         к
+ SAL кT$                         к
+ SAL к                           к
+ SAL ляй                         лй
+ SAL лд(абцдфгйклмопярцужвьы)-   л
+ SAL лT(абцдфгйклмопярцужвьы)-   л
+ SAL лд$                         л
+ SAL лT$                         л
+ SAL л                           л
+ SAL мяй                         мй
+ SAL мд(абцдфгйклмопярцужвьы)-   м
+ SAL мT(абцдфгйклмопярцужвьы)-   м
+ SAL мд$                         м
+ SAL мT$                         м
+ SAL м                           м
+ SAL ояй                         ой
+ SAL од(абцдфгйклмопярцужвьы)-   о
+ SAL оT(абцдфгйклмопярцужвьы)-   о
+ SAL од$                         о
+ SAL оT$                         о
+ SAL о                           о
+ SAL пяй                         пй
+ SAL пд(абцдфгйклмопярцужвьы)-   п
+ SAL пT(абцдфгйклмопярцужвьы)-   п
+ SAL пд$                         п
+ SAL пT$                         п
+ SAL п                           п
+ SAL яяй                         яй
+ SAL яд(абцдфгйклмопярцужвьы)-   я
+ SAL яT(абцдфгйклмопярцужвьы)-   я
+ SAL яд$                         я
+ SAL яT$                         я
+ SAL я                           я
+ SAL ряй<                        рй
+ SAL рд(абцдфгйклмопярцужвьы)-   р
+ SAL рT(абцдфгйклмопярцужвьы)-   р
+ SAL рд$                         р
+ SAL рT$                         р
+ SAL р                           р
+ SAL тяй                         тй
+ SAL тд(абцдфгйклмопярцужвьы)-   т
+ SAL тT(абцдфгйклмопярцужвьы)-   т
+ SAL тд$                         т
+ SAL тT$                         т
+ SAL т                           т
+ SAL уяй                         уй
+ SAL уд(абцдфгйклмопярцужвьы)-   у
+ SAL уT(абцдфгйклмопярцужвьы)-   у
+ SAL уд$                         у
+ SAL уT$                         у
+ SAL у                           у
+ SAL жяй                         жй
+ SAL жд(абцдфгйклмопярцужвьы)-   ж
+ SAL жT(абцдфгйклмопярцужвьы)-   ж
+ SAL жд$                         ж
+ SAL жT$                         ж
+ SAL ж                           ж
+ SAL вяй                         вй
+ SAL вд(абцдфгйклмопярцужвьы)-   в
+ SAL вT(абцдфгйклмопярцужвьы)-   в
+ SAL вд$                         в
+ SAL вT$                         в
+ SAL в                           в
+ SAL ьяй                         ьй
+ SAL ьд(абцдфгйклмопярцужвьы)-   ь
+ SAL ьT(абцдфгйклмопярцужвьы)-   ь
+ SAL ьд$                         ь
+ SAL ьT$                         ь
+ SAL ь                           ь
+ SAL ыяй                         ьй
+ SAL ы(абцдфгйклмопярцужвьы)-    ь
+ SAL ы$                          ь
+ SAL ы                           ы
