*** pt_BR.orig.aff	2010-08-11 22:50:41.000000000 +0200
--- pt_BR.aff	2010-08-11 23:01:57.000000000 +0200
***************
*** 1,3 ****
  SET ISO8859-1
! TRY באדגיךםףץפתüחesianrtolcdugmphbyfvkwjqxz
  
--- 1,3 ----
  SET ISO8859-1
! 
  
***************
*** 13,14 ****
--- 13,32 ----
  
+ NAME    Brazilian Portuguese
+ VERSION 2008-07-07V
+ HOME	http://www.broffice.org/verortografico
+ AUTHOR  Raimundo Santos Moura
+ EMAIL   raimundomoura AT openoffice DOT org
+ AUTHOR  Leonardo Ferreira Fontenelle
+ EMAIL   leo DOT fontenelle AT gmail DOT org
+ COPYRIGHT   LGPL
+ 
+ 
+ FOL  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ LOW  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ UPP  ‗ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞÿ
+ 
+ 
+ MIDWORD	'-.
+ 
+ 
  MAP 6
***************
*** 21,41 ****
  
! BREAK 15
! BREAK BREAK
! BREAK ^-
! BREAK -$
! BREAK ^ex-
! BREAK ^Ex-
! BREAK ^EX-
! BREAK ^recיm-
! BREAK ^Recיm-
! BREAK ^RECÉM-
! BREAK ^pףs-
! BREAK ^Pףs-
! BREAK ^PÓS-
! BREAK ^prף-
! BREAK ^prף-
! BREAK ^PRÓ-
  
  #   Nתmero mבximo de sugestץes       
! MAXNGRAMSUGS 12
  
--- 39,59 ----
  
! #BREAK 15
! #BREAK BREAK
! #BREAK ^-
! #BREAK -$
! #BREAK ^ex-
! #BREAK ^Ex-
! #BREAK ^EX-
! #BREAK ^recיm-
! #BREAK ^Recיm-
! #BREAK ^RECÉM-
! #BREAK ^pףs-
! #BREAK ^Pףs-
! #BREAK ^PÓS-
! #BREAK ^prף-
! #BREAK ^prף-
! #BREAK ^PRÓ-
  
  #   Nתmero mבximo de sugestץes       
! #MAXNGRAMSUGS 12
  
***************
*** 44,46 ****
  
! WORDCHARS -
  
--- 62,64 ----
  
! #WORDCHARS -
  
