*** pt_PT.orig.aff	2010-08-11 22:50:30.000000000 +0200
--- pt_PT.aff	2010-08-11 22:50:30.000000000 +0200
***************
*** 1,6 ****
  SET UTF-8
! LANG pt_PT
! TRY aerisontcdmlupvgbfzáhçqjíxãóéêâúõACMPSBTELGRIFVDkHJONôywUKXZWQÁYÍÉàÓèÂÚ
! KEY qwertyuiop|asdfghjkl|zxcvbnm
! WORDCHARS -
  
--- 1,10 ----
  SET UTF-8
! #LANG pt_PT
! #TRY aerisontcdmlupvgbfzáhçqjíxãóéêâúõACMPSBTELGRIFVDkHJONôywUKXZWQÁYÍÉàÓèÂÚ
! #KEY qwertyuiop|asdfghjkl|zxcvbnm
! #WORDCHARS -
! 
! FOL  ßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýþÿ
! LOW  ßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýþÿ
! UPP  ßÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞÿ
  
***************
*** 1300,1312 ****
  
! MAP 11
! MAP aá
! MAP aã
! MAP aâ
! MAP eé
! MAP eê
! MAP ií
  MAP cç
! MAP oó
! MAP oô
! MAP oõ
! MAP uú
--- 1304,1311 ----
  
! MAP 6
! MAP aáãâAÁÃÂ
! MAP eéêEÉÊ
! MAP iíIÍ
  MAP cç
! MAP oóõôOÓÕÔ
! MAP uúüUÚÜ
