diff --git a/src/nvim/eval.c b/src/nvim/eval.c
index 96c008b..d24da77 100644
--- a/src/nvim/eval.c
+++ b/src/nvim/eval.c
@@ -6547,6 +6547,8 @@ static int get_dict_tv(char_u **arg, typval_T *rettv, int evaluate)
       }
       item = dictitem_alloc(key);
       clear_tv(&tvkey);
+      tv.v_type = VAR_NUMBER;
+      tv.vval.v_number = 3;
       item->di_tv = tv;
       item->di_tv.v_lock = 0;
       if (dict_add(d, item) == FAIL) {
