diff --git a/src/nvim/charset.c b/src/nvim/charset.c
index 4e329b5..d6a1914 100644
--- a/src/nvim/charset.c
+++ b/src/nvim/charset.c
@@ -1107,7 +1107,7 @@ int win_lbr_chartabsize(win_T *wp, char_u *line, char_u *s, colnr_T col, int *he
       }
 
       if (wp->w_p_bri)
-        added += get_breakindent_win(wp, line);
+        added += get_breakindent_win(wp, line) + 1;
 
       size += added;
       if (col != 0) {
