diff --git a/src/nvim/normal.c b/src/nvim/normal.c
index e064d34..e0cf9b5 100644
--- a/src/nvim/normal.c
+++ b/src/nvim/normal.c
@@ -258,11 +258,11 @@ static const struct nv_cmd {
   {'e',       nv_wordcmd,     0,                      false},
   {'f',       nv_csearch,     NV_NCH_ALW|NV_LANG,     FORWARD},
   {'g',       nv_g_cmd,       NV_NCH_ALW,             false},
-  {'h',       nv_left,        NV_RL,                  0},
+  {'h',       nv_right,        NV_RL,                  0},
   {'i',       nv_edit,        NV_NCH,                 0},
   {'j',       nv_down,        0,                      false},
   {'k',       nv_up,          0,                      false},
-  {'l',       nv_right,       NV_RL,                  0},
+  {'l',       nv_left,       NV_RL,                  0},
   {'m',       nv_mark,        NV_NCH_NOP,             0},
   {'n',       nv_next,        0,                      0},
   {'o',       nv_open,        0,                      0},
