*** ms_MY.orig.aff	Wed Aug 31 18:14:01 2005
--- ms_MY.aff	Wed Aug 31 18:14:01 2005
***************
*** 25,26 ****
--- 25,35 ----
  
+ FOL  ßאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ LOW  ßאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ UPP  ßְֱֲֳִֵֶַָֹÊֻּֽ־ֿ׀ׁׂ׃װױײ״ÙÚÛÜÝÞÿ
+ 
+ SOFOFROM abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ßÿְֱֲֳִֵֶַָֹÊֻּֽ־ֿ׀ׁׂ׃װױײ״ÙÚÛÜÝÞ¿
+ SOFOTO   ebctefghejklnnepkrstevvkesebctefghejklnnepkrstevvkeseeeeeeeceeeeeeeedneeeeeeeeeeepseeeeeeeeceeeeeeeedneeeeeeeeeeep?
+ 
+ MIDWORD -
+ 
  PFX B Y 2 
*** ms_MY.orig.dic	Wed Aug 31 18:14:01 2005
--- ms_MY.dic	Wed Aug 31 18:14:01 2005
***************
*** 4939,4941 ****
  datin
- Datoג€™
  datuk/b
--- 4939,4940 ----
